indigo-frontend/src/app/pages/project/project-add/project-add.component.html

```diff
@@ -0,0 +1,50 @@
+<app-modal #addProjectModal title="Add Project" (closed)="close('close')">
+    <div class="w-xl">
+        <form [formGroup]="formGroup" (ngSubmit)="createProject()">
+            <ng-content #modalContent>
+                <app-input label="Project Name" placeholder="Project Name" [formControlName]="'name'"
+                    [required]="true"></app-input>
+                @if (formGroup.get('name')?.hasError('required') && isSubmitted) {
+                <p class="text-red-500 text-sm mt-1">
+                    Email is required
+                </p>
+                }
+
+                <label class="flex flex-col gap-2 my-1" for="keywords">
+                    <span class="text-sm font-medium">Keywords</span>
+                </label>
+                <div class="w-xl p-4 border rounded-lg shadow-md">
+                    <div class="flex flex-wrap gap-2">
```

Comment:
Could you please check if our select component with chips support can be used here. If not, seems that we can move it into separate component.